security-command-center/snippets/src/test/java/vtwo/IamIT.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo;
+
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import com.google.cloud.securitycenter.v2.Source;
+import com.google.cloud.testing.junit4.MultipleAttemptsRule;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import vtwo.iam.GetIamPolicies;
+import vtwo.iam.SetIamPolices;
+import vtwo.iam.TestIamPermissions;
+
+@RunWith(JUnit4.class)
+public class IamIT {
+
+  // TODO(Developer): Replace the below variables.
+  private static final String ORGANIZATION_ID = System.getenv("SCC_PROJECT_ORG_ID");
+  private static final String USER_EMAIL = "someuser@domain.com";
+  private static Source SOURCE;
+  private static final int MAX_ATTEMPT_COUNT = 3;
+  private static final int INITIAL_BACKOFF_MILLIS = 240000; // 4 minutes
+  private static ByteArrayOutputStream stdOut;
+  @Rule
+  public final MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(
+      MAX_ATTEMPT_COUNT,
+      INITIAL_BACKOFF_MILLIS);
```

Comment:
this configuration may result in 12 min of the test execution. it is too long for a single test in the mono repo like `java-docs-samples`. If you cannot run the tests reliable enough please split the tests to multiple files so only a single sample testing will fail in a shorter time

Reply:
I will reduce it to 2 minutes according to FindingsIT samples.
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/security-command-center/snippets/src/test/java/vtwo/FindingsIT.java

Reply:
Two minutes of total failover delays seems reasonable. :+1:

Reply:
I see that the worst case scenario keeps the test running time to 6 minutes.
Let's resolve it for now.
If the tests will prove to be flaky, reducing the total waiting time will be set as a hard requirement.